common/views/components/ManifestContext/ManifestContext.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+import { createContext } from 'react';
+// $FlowFixMe
+const ManifestContext = createContext({});
```

Comment:
Is the reason for the default ManifestContext to be an empty object instead of nullable?

Reply:
No, I've already changed this in the download licensing PR: https://github.com/wellcometrust/wellcomecollection.org/pull/4258/files#diff-d60748032cdad91bdfb73e3f821be159R4

Reply:
Na, but I think we put perf profiling high up as things are getting complicated now and I don't want assume we're not degrading things for people.